src/state/session/index.tsx

```diff
@@ -425,10 +431,10 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
           logger.DebugContext.session,
         )
 
-        logout()
+        clearCurrentAccount()
       }
     })
-  }, [state, logout, initSession])
+  }, [state, clearCurrentAccount, initSession])
```

Comment:
This happens in response to a broadcast channel update. A user could simply be adding a new account, in which case we don't want to nuke their tokens.

If a user _does_ logout, this still still be called, and the other tab will have already nuked their tokens, so same difference.